@xen-orchestra/web-core/lib/components/input/InputLabel.vue

```diff
@@ -0,0 +1,74 @@
+<!-- v2.0 -->
+<template>
+  <div :class="status" class="input-label">
```

Comment:
Why the class name is not `vts-input-label`?
[Guidelines](https://github.com/vatesfr/xen-orchestra/blob/72d60052afefbea28258c760a254e35144ebba56/%40xen-orchestra/web-core/docs/guidelines/component%20definition.md#component-root-elements-class-name-must-be-named-after-the-component-name-prefixed-with-vts-)